components/src/core/components/InfoBox/InfoBox.vue

```diff
@@ -172,6 +172,13 @@ export default defineComponent({
     infoLabel: {
       type: String,
     },
+    height: {
```

Comment:
Can't we give this as Style prop rather than having different props for height and subtitleLine hight.

Reply:
Removed height prop and handled using style prop